waspc/src/Wasp/Generator/Job/Process.hs

```diff
@@ -145,5 +145,5 @@ runNodeCommandAsJob fromDir command args jobType chan = do
         _ -> Nothing
 
     waspNodeRequirementMessage =
-      "Wasp requires node >= " ++ C.nodeVersionAsText ++ " ."
+      "Wasp requires node " ++ C.nodeSemverString ++ " ."
```

Comment:
This fixes [the duplication issue @shayneczyzewski found](https://github.com/wasp-lang/wasp/pull/504#issuecomment-1065237994).